src/BottomNavigation/BottomNavigationButton.js

```diff
@@ -0,0 +1,117 @@
+// @flow weak
+
+import React, { Component, PropTypes, cloneElement, isValidElement } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+import ButtonBase from '../internal/ButtonBase';
+
+export const styleSheet = createStyleSheet('BottomNavigationButton', (theme) => {
```

Comment:
I think that we should add this name under the `MUI_SHEET_ORDER` array.
So that we makes sure we got the right priority to override `ButtonBase` .